pages/roles.js

```diff
@@ -12,14 +12,18 @@ import {
   UncontrolledDropdown,
   DropdownToggle,
   DropdownMenu,
-  DropdownItem
+  DropdownItem,
+  FormGroup,
+  Label,
+  Input
 } from "reactstrap";
 
 class RolesPage extends Component {
   state = {
     users: [],
     newRole: "",
-    userWithNewRole: -1
+    userWithNewRole: -1,
```

Comment:
nit: can we use true/false instead, its a little easier to understand at first glance